internal/environment/lab/lab.go

```diff
@@ -136,14 +132,15 @@ func (l *Lab) Start(ctx context.Context) error {
 func (l *Lab) Close() error {
 	var wg sync.WaitGroup
 	for _, lab := range l.Frontends {
+		log.Debug().Msgf("lab: %v", lab)
```

Comment:
Hmm what is this doing here, add more context to the statement and use fields :)

Reply:
Had some errors that i didn't know where came from, so just used it for analyzing the problem. Removed in next commit